src/pages/generate.tsx

```diff
@@ -116,98 +134,362 @@ export default function ClientIdentifierGenerator() {
+                        Field name: `client_uri`"
+                      />
+                    </Grid>
+                    <Grid container item>
+                      <FormLabel>Redirect URIs</FormLabel>
+                      <Grid container>
+                        <Grid item marginLeft={2} paddingRight={2}>
+                          <FormHelperText>
+                            The URIs that the Solid OIDC Provider is allowed to
+                            redirect to after the user authenticated at the OIDC
+                            Provider.
+                          </FormHelperText>
+                          <FormHelperText>
+                            Flow: Your application will send an authentication
+                            request to the OIDC Provider with one redirect URI.
+                            The OIDC Provider redirects the browser/user agent
+                            to the redirect URI after the user authenticated and
+                            issues a code to your application that it can use to
+                            complete authentication. Field name: `redirect_uris`
+                          </FormHelperText>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+
+                    <Grid item container>
+                      <FieldArray
+                        name="redirectUris"
+                        component={RedirectUrisComponent}
+                      />
+                    </Grid>
+                    <Grid container item>
+                      <FormControlLabel
+                        label="Support usage of refresh tokens / offline access"
+                        control={
+                          <Checkbox
+                            name="useRefreshTokens"
+                            value={form.values.useRefreshTokens}
+                            checked={form.values.useRefreshTokens}
+                            onChange={form.handleChange}
+                          />
+                        }
+                      />
+                      <Grid item marginLeft={2} paddingRight={2}>
+                        <FormHelperText>
+                          A refresh token can be used to request new access
+                          tokens after they expire. If not requested, the user
+                          will have to re-authenticate after the access token
+                          expires. Field names: The field `scope` will have the
+                          value `offline_access` set and the field `grant_types`
+                          will have the value `refresh_token` set.
+                        </FormHelperText>
+                      </Grid>
+                    </Grid>
+                    <Grid container item>
+                      <Accordion>
+                        <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                          More fields
```

Comment:
```suggestion
                          Additional information displayed to users
```
I'm afraid "more fields" is too generic